repos/tap/actions/files/local/setActiveFile.js

```diff
@@ -24,12 +24,9 @@ const getScreen = screenId => {
  */
 export const setActiveFile = (fileModel, content, screenId) => {
   const screenModel = getScreen(screenId)
-
-  // If no screen is found, then we can't set the file
-  // So log an error and return
   if(!screenModel) return devLog(`error`, `Can not find screen from id`, screenId)
-  
-  const updatedFile = { ...fileModel, modified: content || fileModel.modified }
+  const updatedFile = fileModel
+  content && (updatedFile.content = content)
```

Comment:
Pretty sure there should be no situations where we update the content locally
* Updates to a file should only come from load the file from the backend
* So we should get rid of this, unless theres something I'm missing here

Reply:
the use case currently is only right after a file gets saved.
then it also updates the currently active File content.
either that or we call the backend API right after save is successful

Reply:
Why don't we just return the saved file from the API after is saved?
* Save the file, then in the response send back the updated fileModel
* Then on the frontend, update the file in both activeFile, and it's other fileType store. It has to be both, or there will be issues